cpp/ps0001/main.cpp

```diff
@@ -4,17 +4,21 @@ using namespace std;
 
 int main()
 {
+  // prompting user for 3 numers to add 1/2
+  unsigned short num1;
+  cout << "enter value of a " <<endl;
+  cin >> num1;
+  unsigned short num2;
+  cout << "enter value of b " <<endl;
+  cin >> num2;
+  unsigned short num3;
+  cout << "enter value of c " <<endl;
+  cin >> num3;

```

Comment:
- [ ] declared 3 variables to store user inputs with data type `unsigned short int`
- [x] used `std::cin` in a right way to prompt user for 3 numbers to add